### DISCUSSION TITLE
How to get duration of time between 2 dates?

### DISCUSSION THREAD

**Discussion by Sandy381** [fbaligand/kibana-enhanced-table#311] [General]
Hi ,

I'm trying to get duration between 2 dates - difference of 2 dates "updated" and "created".

When I take difference between 2 dates , i get a number.
If I select date, it shows some odd date not relating.

But I want to show duration i.e 10 sec, 1 hour, 1 day, 1 month.

![Screenshot 2023-02-12 at 2 13 04 AM](https://user-images.githubusercontent.com/74658016/218280442-2eaf1d61-694c-45f0-aaa2-42211817acd5.png)

How do i Accomplish this?

**Comment by fbaligand** [comment]
Well, the difference between 2 dates is the millisecond count duration.
If your duration is less than 24h, you can display it as a date and show only hours, minutes, seconds and milliseconds. 
if you want to show durations greater than 24h, you have to use a string format and use a complex formula that computes day count, hour count, minute count, …
There is no “out of the box” way to do that.
